tests/http2-disabled/test.yaml

```diff
@@ -0,0 +1,28 @@
+requires:
+  features:
+    - HAVE_LIBJANSSON
+  min-version: 7.0.0
```

Comment:
The Suricata fix's backport has been merged into master-6.0.x, should the min-version here be updated?